### ISSUE TITLE
Confused about Function Editor

### ISSUE THREAD

**Issue by ladlon** [turtletooth/tahoma2d#336]
If I select a column, and then select Function Editor in the right mouse button menu, I was assuming it would open up the editor, showing only that column.

Instead, it opens the editor, showing every item in the scene, with the camera selected.

Am I doing something wrong?  If not, wouldn't it be better/more logical for it to only display the selected item, so you don't have to fish through the entire list of scene items every single time?

When you just open the Function Editor on its own, sure... have everything shown.

What am I misunderstanding?

**Comment by artisteacher** [comment]
I don't think you are misunderstanding anything. I think the initial design of how keyframes are shown/hidden has always been flawed.

I really like the idea of the function editor opening with only the active column visible. For the timing room, the Function Editor could also use a context menu item to "show selected layer only" in addition to the "show animated only" or "show all" commands.

Eventually, it would also be helpful to show/hide the columns & etc. with a visibility toggle - just like how it works in the Timeline. I don't like the behavior of clicking on the folder icons to enable/disable visibility. Sometimes you just want to look inside.
